internal/bminventory/inventory.go

```diff
@@ -6751,3 +6782,16 @@ func getDefaultHighAvailabilityAndMasterCountParams(highAvailabilityMode *string
 	// both are set
 	return highAvailabilityMode, controlPlaneCount
 }
+
+func extractMirroredRegistriesFromConfig(log logrus.FieldLogger, mirrorRegistryConfig *common.MirrorRegistryConfiguration) []string {
+	var ignorableRegistries []string
+	mirroredRegistries, err := mirrorregistries.ExtractLocationMirrorDataFromRegistriesFromToml(mirrorRegistryConfig.RegistriesConf)
```

Comment:
What happens if extraction fails?

Reply:
We should just put a comment here to acknowledge that we are happy this is handled elsewhere

Reply:
Added a comment! What do you think?